content_scripts/desktopCamera.js

```diff
@@ -462,13 +463,13 @@ createNameSpace('realityEditor.device.desktopCamera');
 
     function panToggled() {
         if (cameraTargetIcon) {
-            cameraTargetIcon.visible = knownInteractionStates.pan || knownInteractionStates.rotate || knownInteractionStates.scale;
+            //cameraTargetIcon.visible = knownInteractionStates.pan || knownInteractionStates.rotate || knownInteractionStates.scale;
```

Comment:
Is there a reason you toggled these off? I think they can be a helpful visual feedback while rotating and panning

Reply:
That makes sense, turning off the blue cube in the middle 👍 .

But I think you also toggled off the 2D icons that move with the cursor. There's one that's semi-transparent, showing where you started the gesture, and another that moves with the mouse.

![pan-cursor-icon](https://user-images.githubusercontent.com/32580292/228878552-d32a9669-273c-4253-9421-c2093a25be57.png)